sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -45,7 +45,17 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
 
 ## Tasks
 
-In this app, users will have access to a **Spotlight**, **Gallery**, **Favorite**, and **Detail** page.
+This app will include four main pages: **Spotlight**, **Gallery**, **Favorite**, and **Detail**.
+
+Each page will need access to the information from the API. Consider where to fetch and store the art piece data to ensure that all pages can easily access it.
+
+<details>
+<summary>Hint ✨</summary>
+In Next.js, the place to store data or states that need to be accessed across the app is in <code>_app.js</code>.
+
+It's best practice to consolidate related information in a single state. However, if you're not comfortable with this approach, you can use separate states. Keep in mind that you’ll need to identify which information corresponds to each art piece, so consider saving the slug or a unique identifier in each state to maintain clarity.
```

Comment:
I would keep the solution open without ranking them, this would lead to students to overwhelm themselves I guess. 
````suggestion
There are multiple ways to store the local information of your art pieces:
1. Single "info" state:
```js
const carInfo = useState([
  {carId: '1234' , isStarred: false, comments: []},
  {carId: '5678' , isStarred: true, comments: []},
])
```
2. Separate States per information:
```js
const starredCarIds = useState(["1234","5678"])
const comments = useState({
  id: '1',
  carId: '1234',
  comment: 'comment 1'
},{
    id: '2',
  carId: '1234',
  comment: 'comment 2'
}])
```
````

Reply:
Nice, I combine it with the one I did.
Let me know.
I move it of place to be near where the student may need this hint.